src/server/symbols.map

```diff
@@ -1458,3 +1458,9 @@ global:
   };
 } MIR_SERVER_INTERNAL_2.19;
 
+MIR_SERVER_INTERNAL_2.20.2 {
+global:
+  extern "C++" {
+    mir::Server::override_the_pointer_input_dispatcher*;
+  };
+} MIR_SERVER_INTERNAL_2.20;
```

Comment:
And this, need a 2.21 stanza.

Reply:
That depends on whether we're rolling a 2.20.2?